src/api/viya/saveLog.ts

```diff
@@ -0,0 +1,50 @@
+import { Job } from '../..'
+import { RequestClient } from '../../request/RequestClient'
+import { fetchLog } from '../../utils'
+import { WriteStream } from 'fs'
+import { writeStream } from './writeStream'
+
+export async function saveLog(
```

Comment:
does this need a JSDoc comment?

Reply:
There's no real need since this function is only used internally (the export is just so it's available within other internal modules) and not available for outside use. But I've added a JSDoc comment anyway.